packages/wp-story-editor/src/index.js

```diff
@@ -35,38 +38,42 @@ import { initializeTracking } from '@web-stories-wp/tracking';
 /**
  * Internal dependencies
  */
+import * as apiConfig from './api';
```

Comment:
Overall I like splitting up the API functions like that 👍 

It's not as low-level as the `dataAdapter` approach in `packages/dashboard/src/app/api/apiProvider.js`.

So once this PR makes it in, we should consider refactoring the dashboard in a similar way.